adoc/retail_chap_formulas.adoc

```diff
@@ -0,0 +1,209 @@
+
+This configuration allows for the branch server to provide DNS, TFTP, PXE and FTP services to terminals, which are configured through {smr} formulas in the {susemgr} {webui}.
+Optionally, the branch server can also provide DHCP services in this configuration.
+
+[NOTE]
+====
+If DHCP services are not provided by the branch server, ensure that your external DHCP configuration is set correctly:
+* The [systemitem]``next-server`` option must point to the branch server for PXE boot to work
+* The [systemitem]``filename`` option must correctly identify the network boot program (by default, this is [path]``/boot/pxelinux``)
+* The [systemitem]``domain-name-servers`` option must point to the branch server for correct host name resolution
+====
+
+
+.Procedure: Setting up a branch server with a shared network
+
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
+. In the [guimenu]``Branch Network`` section, set these parameters:
+* Keep [guimenu]``Dedicated NIC`` unchecked
+* Select which services to enable on the branch server’s firewall.
+Ensure you include DNS, TFTP and FTP services.
+* Select the [guimenu]``bind`` DNS forwarder mode.
+* Check DNS forwarder fallback if you want to rely on an external DNS if the branch DNS fails.
+* Specify the working directory, and the directory owner and group.
+. Click btn:[Save] to save your changes.
+. Apply the highstate.
+
+
+
+[[retail.sect.formulas.image-sync]]
+== Image Sync Formula
+
+The image sync formula is used to configure how images are synchronized to the branch server.
+
+
+.Procedure: Configuring image synchronization
+
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
+. Select the [systemitem]``Pxe`` formula, and click [btn]``Save``.
+. Navigate to the menu:Formulas[Pxe] tab, and set these parameters:
+* In the [guimenu]``Kernel filename`` field, keep the default value.
+* In the [guimenu]``Initrd filename`` field, keep the default value.
+* In the [guimenu]``Kernel commandline parameters`` field, keep the default value.
+* In the [guimenu]``PXE root directory`` field, enter the path to the saltboot directory (for example, [systemitem]``/srv/saltboot``).
+* In the [guimenu]``Branch id`` field, type a name to use as a branch identifier (for example: [systemitem]``Branch0001``).
+Use only alphanumeric characters for the branch identifier.
+. Click [btn]``Save Formula`` to save your configuration
+. Apply the highstate.
+
+
+
+[[retail.sect.formulas.pxe]]
+== PXE Formula
+
+The PXE formula is used to configure PXE booting on the branch server.
+
+
+.Procedure: Configuring PXE booting
+. Step One
+. Step Two
+. Another step
+////
+Kernel filename:
+Initrd filename:
+Kernel commandline parameters:
+PXE root directory:
+Branch id:
+////
+
+
+
+[[retail.sect.formulas.saltboot]]
```

Comment:
Just note that this is duplicate information with [Configure the Saltboot formula in administration chapter](https://github.com/SUSE/doc-susemanager/blob/develop/adoc/retail_chap_admin.adoc#configure-the-saltboot-formula).
However this piece of information is very valid to be in administration chapter as well as it will be used during regular administration. How do we solve this duplicity? Just make a link from administration chapter to here?

Reply:
I think all formula info should be in the formula chapter. If we mention a particular formula elsewhere, we can xref to the formula chapter, but I wouldn't create a section just for that purpose.

Reply:
Hrm, these two procedures are quite different. Which one is the most accurate?

Reply:
@aaannz We need your guidance on this, please :)

Reply:
The procedure here named `.Procedure: Configuring the branch server with saltboot` is nonsense. Saltboot formula is about disks and partitioning and selecting image.

Procedure below is wrongly named as saltboot, but is in fact Bind formula.